src/controllers/cardController.ts

```diff
@@ -1,67 +0,0 @@
-import { Request, Response } from "express";
-import { ActivateCardData, InfoCardInterface } from "../interfaces/Card";
-import * as cardService from "../services/cardService";
-
-async function postCard(req: Request, res: Response) {
-    const {
-        employeeId,
-        type,
-    }: InfoCardInterface = req.body;
-
-    const card = await cardService.postNewCard({ employeeId, type });
-
-    return res.send(card);
```

Comment:
Lembre de retornar o status mesmo se for só pra o retorno de informações. Nesse caso, retornando o card, você poderia retornar algo similar a isso:

``return res.status(200).send(card)``